app/controllers/home.js

```diff
@@ -10,24 +10,37 @@ export default Ember.Controller.extend({
     showNotSavedModal: false,
     published: false,
     saved: false,
+    isOpen: Ember.computed('node', async function(){ 
+        let node = await this.get('model.node')
+        console.log(this.get('model.node'))
+        this.set('isOpen', !node.get('public') )
+
+        $( document ).ready(function() {
+            $('.alert-message').css('display' , 'block')
+        });
+    }),
     unpublishedChanges:  Ember.computed('editMode', function() {
         return this.store.findRecord('home', this.get('model.guid')).then((record)=>{ 
-            
+
             let unpublishedPageData = record.get('unpublishedPageData');
             let pageData =  record.get('pageData');
 
             if(_.isEqual(unpublishedPageData, pageData)){
                 this.set('unpublishedChanges', false)
             }else{
                 this.set('unpublishedChanges', true)
-
             }
         });
     }),
     savedPageData: "",
     isAdmin: Ember.computed('node', function(){
         this.send('canUserEdit')
-        return this.get('model.node.currentUserPermissions').includes('admin');
+        if(this.get('model.node') !== null){    
+            return this.get('model.node.currentUserPermissions').includes('admin');
+        }else{
```

Comment:
It's up to you but you don't need to write out else, you can just return here. If the stament is true your next return will not run.

```js
if(this.get('model.node') !== null){    
    return this.get('model.node.currentUserPermissions').includes('admin');
}
return false;

```